.github/workflows/snuba-integration-test-py3.6.yml

```diff
@@ -0,0 +1,67 @@
+name: snuba integration [py3.6]
```

Comment:
I'd prefix the file name with `backend-`.

Reply:
Hmm do we think that's needed? Seems a bit redundant to me

Reply:
yeah I was kind of aiming towards that with the file names, but in a different order. I prefer the filename w/ the order you described, but I wanted the file names to match the workflow names and 
 I'd prefer to leave `backend` out of the workflow name because it may get difficult to scan for what you want when we start porting in the travis suites...

![image](https://user-images.githubusercontent.com/79684/96311383-2db56e80-0fbe-11eb-881f-202f0cdfcde3.png)

We're going to have 3x backend2.7, 1x snuba 2.7 - would symbolicator and relay also be considered backend?

Reply:
> would symbolicator and relay also be considered backend?

Yeah.

Reply:
ok, once we have all our tests in GHA, I'll do some mock ups and take a poll